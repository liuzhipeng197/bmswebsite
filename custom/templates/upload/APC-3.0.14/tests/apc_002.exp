object(foo)#%d (0) {
}
object(foo)#%d (0) {
}
object(foo)#%d (1) {
  ["a"]=>
  bool(true)
}
===DONE===