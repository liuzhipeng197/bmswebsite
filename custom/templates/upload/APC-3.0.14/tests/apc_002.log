
---- EXPECTED OUTPUT
object(foo)#%d (0) {
}
object(foo)#%d (0) {
}
object(foo)#%d (1) {
  ["a"]=>
  bool(true)
}
===DONE===
---- ACTUAL OUTPUT
PHP:  Error parsing /opt/bms/custom/templates/upload/APC-3.0.14/tmp-php.ini on line 1231
object(foo)#1 (0) {
}
object(foo)#1 (0) {
}
object(foo)#1 (1) {
  ["a"]=>
  bool(true)
}
===DONE===
---- FAILED
