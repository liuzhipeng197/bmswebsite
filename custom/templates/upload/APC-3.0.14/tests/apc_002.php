<?php

class foo { }
$foo = new foo;
var_dump($foo);
apc_store('foo',$foo);
unset($foo);
$bar = apc_fetch('foo');
var_dump($bar);
$bar->a = true;
var_dump($bar);

?>
===DONE===
