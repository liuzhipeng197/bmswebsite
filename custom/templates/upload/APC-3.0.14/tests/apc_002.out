PHP:  Error parsing /opt/bms/custom/templates/upload/APC-3.0.14/tmp-php.ini on line 1231
object(foo)#1 (0) {
}
object(foo)#1 (0) {
}
object(foo)#1 (1) {
  ["a"]=>
  bool(true)
}
===DONE===